googleapiclient/discovery.py

```diff
@@ -82,6 +82,9 @@
 VARNAME = re.compile('[a-zA-Z0-9_-]+')
 DISCOVERY_URI = ('https://www.googleapis.com/discovery/v1/apis/'
```

Comment:
On Tue, Mar 15, 2016 at 6:59 PM, Nathaniel Manista <notifications@github.com
> wrote:

> In googleapiclient/discovery.py
> <https://github.com/google/google-api-python-client/pull/202#discussion_r56272232>
> :
>
> > @@ -82,6 +82,9 @@
> >  VARNAME = re.compile('[a-zA-Z0-9_-]+')
> >  DISCOVERY_URI = ('https://www.googleapis.com/discovery/v1/apis/'
>
> It's a part of the public API, so we can't remove it until we bump to
> 2.0.0, regardless of the fact that we ourselves are no longer using it.
>
> Always strive for a minimal public API in a software library. :-\
>

Yes. Thanks for the review.

> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/google/google-api-python-client/pull/202/files/12b7cd39c4609c641e510499eb0ec1671a909a92#r56272232>
>